projects/components/src/skeleton/skeleton.component.ts

```diff
@@ -0,0 +1,85 @@
+import { ChangeDetectionStrategy, Component, Input, OnChanges } from '@angular/core';
+
+@Component({
+  selector: 'ht-skeleton',
+  template: `
+    <ng-container *ngFor="let k of this.numberOfIterations">
+      <ng-container [ngSwitch]="this.shapeStyle">
+        <div *ngSwitchCase="'${SkeletonType.RectangleText}'" [ngClass]="this.containerClass"></div>
+        <div *ngSwitchCase="'${SkeletonType.Circle}'" [ngClass]="this.containerClass"></div>
+        <div *ngSwitchCase="'${SkeletonType.Square}'" [ngClass]="this.containerClass"></div>
+        <div *ngSwitchCase="'${SkeletonType.TableRow}'" [ngClass]="this.containerClass"></div>
+        <div *ngSwitchCase="'${SkeletonType.Donut}'" [ngClass]="this.containerClass">
+          <div class="donut-inner"></div>
+        </div>
+        <div *ngSwitchCase="'${SkeletonType.ListItem}'" [ngClass]="this.containerClass">
```

Comment:
Could you please change the switch case to contain:
1. TableRow
2. ListItem
3. Default template for all other